docker/start.py

```diff
@@ -83,7 +89,7 @@ def generate_config_from_template(config_dir, config_path, environ, ownership):
 
     # Convert SYNAPSE_NO_TLS to boolean if exists
     if "SYNAPSE_NO_TLS" in environ:
-        tlsanswerstring = str.lower(environ["SYNAPSE_NO_TLS"])
+        tlsanswerstring = str.lower(environ["SYNAPSE_NO_TLS"])  # type
```

Comment:
What does "# type" do?

Reply:
Nothing. I think this was a `#type: ignore` I started adding but thought better of it. Will remove, thanks!

Reply:
https://github.com/matrix-org/synapse/pull/12528/commits/283d9614018a154f0570a43d2b2dde4dfac52b4c